clarifai/workflows/validate.py

```diff
@@ -0,0 +1,67 @@
+from schema import And, Optional, Regex, Schema, SchemaError, Use
+
+# Non-empty, up to 32-character ASCII strings with internal dashes and underscores.
+_id_validator = And(str, lambda s: 0 < len(s) <= 32, Regex(r'^[0-9A-Za-z]+([-_][0-9A-Za-z]+)*$'))
+
+# 32-character hex string, converted to lower-case.
+_hex_id_validator = And(str, Use(str.lower), Regex(r'^[0-9a-f]{32}'))
+
+
+def _model_does_not_have_model_version_id_and_other_fields(m):
```

Comment:
Can we name the function anything simpler rather than the whole Description?

Reply:
Good Point. But I feel in `validate.py` the long name is needed so as to be more specific on functionality. However we can make function names in `workflows/utils.py` much shorter. Let me do that.